locale/lang-es.js

```diff
@@ -178,14 +178,14 @@ SnapTranslator.dict.es = {
 */
 
     // translations meta information
-    'language_name':
-        'Espa\u00F1ol', // the name as it should appear in the language menu
-    'language_translator':
-        'V\u00EDctor Manuel Muratalla Morales / Cristi\u00E1n Rizzi Iribarren / Alfonso Ruzafa / David Mart\u00EDn', // your name for the Translators tab
-    'translator_e-mail':
-        'victor.muratalla@yahoo.com / rizzi.cristian@gmail.com', // optional
-    'last_changed':
-        '2023-11-20', // this, too, will appear in the Translators tab
+    "language_name":
```

Comment:
I'm not sure these even belong in the current format of the translation file, but using `"` made it easier to parse the file and extract the translations.